### DISCUSSION TITLE
Redundancy of management service?

### DISCUSSION THREAD

**Discussion by ser** [netbirdio/netbird#834] [Issue Triage]
Is it possible to have redundancy of self hosted management service in any way? I observed when management service is disconnected, all peers momentary disconnect as well.

**Comment by braginini** [comment]
Hey @ser 
There is no redundancy supported yet. We will be working on it soon. 

Peers should not disconnect from each other even if the Management service is down. Could you please elaborate on this? What do you see when running:
```
netbird status -d
```

**Comment by ser** [comment]
Sorry, I suppose I was wrong :) It's signal service which breaks down connections. How do I make signal service redundant?

**Reply by mlsmaycon** [reply]
Hello @ser can you share the client logs?

In Unix-like they are located at /var/log/netbird/client.log
In Windows: C:\\ProgramData\\Netbird\\client.log

Besides that, can you confirm if you are running on self-hosted and if there is a reverse proxy in front of the signal service?

**Comment by braginini** [comment]
Hey @ser 
Not Signal nor Management should cause disconnects of the agents when unavailable. We will check this out.
